spec/stat_tracker_spec.rb

```diff
@@ -1,10 +1,51 @@
-require 'spec_helper'
+require '../futbol/lib/stat_tracker'
+require 'rspec/mocks/standalone'
 
-describe StatTracker do
-  it "exists" do
-stat_tracker = StatTracker.new(data)
+RSpec.describe StatTracker do
+  before(:each) do
```

Comment:
should we use mock/stub or give the rspec file the info from the runner?

Reply:
Hmmm I'd like to here more of what you mean Zanna. But I like the use of mocks here in our personal RSpec tests since we can always test against the existing RSpec inside of the spec harness which has the CSV info.

Reply:
That's true, I was talking about just declaring the file paths in the rspec but we can keep it as a mock/stub for testing :)

Reply:
Ah okay, thanks for responding!